apps/content/api/internal/recitation_list.py

```
@@ -0,0 +1,53 @@
+
+class RecitationFilter(FilterSchema):
+    publisher_id: list[int] | None = Field(None, q="resource__publisher_id__in")
+    reciter_id: list[int] | None = Field(None, q="reciter_id__in")
+    riwayah_id: list[int] | None = Field(None, q="riwayah_id__in")
+
+
+@router.get("recitations/", response=list[RecitationListOut])
+@paginate
+@ordering(ordering_fields=["name", "created_at", "updated_at"])
+@searching(search_fields=["name", "description", "resource__publisher__name", "reciter__name"])
+def list_recitations(request, filters: RecitationFilter = Query()):
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)